spec/integration/manifest.spec.yml

```diff
@@ -237,7 +237,7 @@ spec:
       additionalProperties: false
       properties:
         github:
-          description: GitHub repository name of package maintainer.
+          description: Package maintainer GitHub team name.
```

Comment:
Another option for this description:
- Github team name of the package maintainer